README.md

```diff
@@ -25,6 +23,12 @@ Create a database and the versioning function:
 ```sh
 createdb temporal_test
 psql temporal_test < versioning_function.sql
+psql temporal_test < system_time_function.sql
+```
+
+If you would like to have `set_system_time` function available (more details [below](#system-time)) you should run the following as well:
+```sh
+psql temporal_test < system_time_function.sql
```

Comment:
I don't understand this passage. Why are we duplicating the instruction?

Reply:
Whoops, it seems like a copy/paste issue. Fixed.